### ISSUE TITLE
Saying my login is incorrect

### ISSUE THREAD

**Issue by ImProvolone** [RumbleMike/ValorantStreamOverlay#134]
Says my login is incorrect but I know I put the password in correctly

**Comment by ImProvolone** [comment]
![image](https://user-images.githubusercontent.com/77126251/103944632-f4deee80-5101-11eb-9d15-799699c9a871.png)

**Comment by johnalbert-dot-py** [comment]
Please wait for new program to upload. The devs just update the code and maybe they release later.

**Comment by johnalbert-dot-py** [comment]
I did encounter the error too (on my own app based on [ValorantRankedPoints Project](https://github.com/RumbleMike/ValorantRankedPoints/tree/master/GetRankedPoints)) and the devs just fix it the way I fix the bug in may app.

**Comment by DarthBring** [comment]
yeah, i got the same problem with the login info, so i just need to wait T_T

**Comment by johnalbert-dot-py** [comment]
Don't worry guys, the devs already fixed the code and maybe release for a minute I think? hahahha

**Comment by DarthBring** [comment]
the problem with me wasnt the client, it just wasnt updated so i dowloaded it again and problem fixed. ty
